mobile-app/src/screens/HomeScreen.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { View, Text, Button, TextInput } from 'react-native';
+import { HomeScreenProps } from './navigationTypes';
+
+export function HomeScreen({ navigation }: HomeScreenProps) {
+  return (
+    <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
+      <Text style={{ fontSize: 32 }}>Hex Game</Text>
+      <Button title='Create local game' onPress={() => navigation.navigate('Local')}>
+      </Button>
+      <Text>Or</Text>
+      <Button title='Create online game' onPress={() => { }}>

```

Comment:
In a real world, you never have to deploy something which is not ready. For 3 reasons :

- Everything merged on master should works. Because everything on master can be deployed every time.
- You will develop a smaller PR :)
- Your PR actually have dead code. Imagine what can happen if your PM suddenly change his mind ? Your dead code can be forgotten or you have to remove it first.

You should display only one button which works well and do not implements others.

Obviously in this context it's ok and you can merge but in a real project, I think it's important to never implement something which could work in an hypothetical future.